build.rs

```diff
@@ -6,9 +8,14 @@ fn main() {
     {
         #[cfg(target_os = "macos")]
         {
+            let probelib = if std::path::Path::new(MAC_FUSE_4).exists() {
```

Comment:
Checking for the existence of this path seems brittle to me. Is it documented anywhere that it's guaranteed to exist for MacFuse 4.0? I think it would be better to detect this via pkg_config. In other words, first try to link against "fuse" and then fallback to "osxfuse", just like the non-macos code path below: https://github.com/cberner/fuser/pull/104/files#diff-d0d98998092552a1d3259338c2c71e118a5b8343dd4703c0c7f552ada7f9cb42L23

Reply:
my reasoning was that some package management tools might do some symlink shenanigans with the pkg_config files,  so I decided to test for a file that only v4 installs. But you are totally right, simply trying to use it is a more stable approach - I implemented your suggested changes.